decdk.specs.json

```diff
@@ -1,5 +1,5 @@
 {
-  "schemaVersion": "2.0.0",
+  "schemaVersion": "3.0.0",
```

Comment:
For now, while we haven't published the docs, we don't need to worry about this.